File: _study/COS_PRO_2/cpp/stage2/8.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int solution(int number) {
    int count = 0;
    while(number >= 0){
        int n = number % 10;
        if (n == 2 || n == 3 || n == 5 || n == 7)
            count += 1;
        number /= 10;
        if (number <= 0) break;
    }
    return count;
}

// 아래는 테스트케이스 출력을 해보기 위한 main 함수입니다. main함수는 잘못된 부분이 없으니, solution함수만 수정하세요.
int main() {
    int number = 29022531;
    int ret = solution(number);

    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    cout << "solution 함수의 반환 값은 " << ret << " 입니다." << endl;
}

File: _study/COS_PRO_2/python/stage3/8.py
# -*- coding: utf-8 -*-
# UTF-8 encoding when using korean

def solution(programs):
    answer = 0
    used_tv = [0] * 25

    for program in programs:
        for i in range(program[0], program[1]):
            used_tv[i] = used_tv[i] + 1

    for i in used_tv:
        if i >= 2:
            answer = answer + 1
    return answer


programs = [[1, 6], [3, 5], [2, 8]]
ret = solution(programs)

print("solution 함수의 반환 값은", ret, "입니다.")
